collectors/prometheus.go

```diff
@@ -101,9 +104,12 @@ func NewPrometheusExporter(cfg *PrometheusConfig, lnd *lndclient.LndServices,
 			NewPeerCollector(lnd.Client, errChan),
 			NewInfoCollector(lnd.Client, errChan),
 		},
-		htlcMonitor.collectors()...,
 	)
 
+	if !monitoringCfg.DisableHtlc {
```

Comment:
The linter complains that because we deleted the `htlcMonitor.collectors()...,` from the `collectors := append()` above, we don't actually append anything anymore. So that needs to be re-written.